src/main/java/com/google/codeu/servlets/mapsServlet.java

```diff
@@ -0,0 +1,83 @@
+package com.google.codeu.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import com.google.gson.JsonArray;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.Scanner;
+
+/**
+ * Returns bubble tea data as a JSON array, e.g. [{"lat": 38.4404675, "lng": -122.7144313}]
+ */
+@WebServlet("/boba-final")
+public class mapsServlet extends HttpServlet{
```

Comment:
1. Name the servlet after the endpoint. (The endpoint here is "/boba-final".)
2. Use camel-case (e.g. MapsServlet, not mapsServlet). After you rename, make sure that your filename has the exact same name as the Java class. If you're having trouble renaming, you might need to rename the file to another name (e.g. "temp"), then rename it back.